From e5f0684928fb054d32b058e84768e6539544ee12 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 29 Nov 2024 18:29:49 +0100
Subject: [PATCH 1516/1695] plat-stm32mp1: remove unused function
 stm32mp_pm_retram_resume_ep

Remove the unused function stm32mp_pm_retram_resume_ep().

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git fcda1d0961370ffc7a2e48a231a4cac6a0ee7757 plat-stm32mp1: remove unused function stm32mp_pm_retram_resume_ep]
Fixes: cf483e4784e5 ("plat-stm32mp1: add low power drivers for stm32mp1")
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/423278
Change-Id: I947403ae3b654cccd10e969ea2e4672b6b3050d4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/437074
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp1/pm/context.c | 7 -------
 core/arch/arm/plat-stm32mp1/pm/context.h | 1 -
 2 files changed, 8 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp1/pm/context.c b/core/arch/arm/plat-stm32mp1/pm/context.c
index b4ea04852..cfc91404b 100644
--- a/core/arch/arm/plat-stm32mp1/pm/context.c
+++ b/core/arch/arm/plat-stm32mp1/pm/context.c
@@ -221,13 +221,6 @@ static void restore_time(void)
 #endif
 }
 
-uintptr_t stm32mp_pm_retram_resume_ep(void)
-{
-	struct retram_resume_ctx *ctx = get_retram_resume_ctx();
-
-	return (uintptr_t)&ctx->resume_sequence;
-}
-
 /* Clear the content of the PM mailbox */
 void stm32mp_pm_wipe_context(void)
 {
diff --git a/core/arch/arm/plat-stm32mp1/pm/context.h b/core/arch/arm/plat-stm32mp1/pm/context.h
index 1ce85b1f6..5e5a3cbd2 100644
--- a/core/arch/arm/plat-stm32mp1/pm/context.h
+++ b/core/arch/arm/plat-stm32mp1/pm/context.h
@@ -95,7 +95,6 @@ int stm32mp1_set_pm_domain_state(enum stm32mp1_pm_domain domain, bool status);
 uint32_t stm32mp1_get_lp_soc_mode(uint32_t psci_mode);
 int stm32mp1_set_lp_deepest_soc_mode(uint32_t psci_mode, uint32_t soc_mode);
 
-uintptr_t stm32mp_pm_retram_resume_ep(void);
 bool stm32mp1_is_retram_during_standby(void);
 
 int stm32mp_ccm_encrypt_teeram(struct retram_resume_ctx *ctx,
-- 
2.47.2

